rules/legacy/tests/Rector/Include_/AddTopIncludeRector/AddTopIncludeRectorTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Legacy\Tests\Rector\Include_\AddTopIncludeRector;
+
+use Iterator;
+use Rector\Core\Testing\PHPUnit\AbstractFileSystemRectorTestCase;
+use Rector\Legacy\Rector\Include_\AddTopIncludeRector;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class AddTopIncludeRectorTest extends AbstractFileSystemRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function test(SmartFileInfo $fileInfo): void
+    {
+        $this->doTestFileInfo($fileInfo);
+    }
+
+    public function provideData(): Iterator
+    {
+        return $this->yieldFilesFromDirectory(__DIR__ . '/Fixture');
+    }
+
+    protected function getRectorsWithConfiguration(): array
+    {
+        return [
+            AddTopIncludeRector::class => [
+                '$settings' => [
+                    'type' => 'TYPE_INCLUDE',
+                    'file' => '__DIR__ . "/../autoloader.php"',
+                ],
+            ],
```

Comment:
Ah, I see it now. Array makes sense only in case of multiple instances, like rename these 30 classes to these 30 classes.

With this 1-level setup, you can use them directly:

```suggestion
            AddTopIncludeRector::class => [
                '$type' => 'TYPE_INCLUDE',
                '$file' => '__DIR__ . "/../autoloader.php"',
            ],
```

Reply:
OK, that is what I was trying to figure out, but could not seem to get it to work.  Will change.